src/System.Reflection.Metadata/ref/System.Reflection.Metadata.cs

```diff
@@ -99,6 +99,9 @@ public partial struct AssemblyDefinition
         public System.Reflection.Metadata.StringHandle Name { get { throw null; } }
         public System.Reflection.Metadata.BlobHandle PublicKey { get { throw null; } }
         public System.Version Version { get { throw null; } }
+#if !NETSTANDARD11
+        public System.Reflection.AssemblyName GetAssemblyName() { throw null; }
```

Comment:
This was my mistake to help her fix the AllConfigurations build errors, but from what I just checked they are available but in a reference that was not included when building for ns1.1. I will work with her to get this fixed.